src/java/org/apache/cassandra/index/Index.java

```diff
@@ -513,21 +513,24 @@ public Indexer indexerFor(DecoratedKey key,
          * Notification of the start of a partition update.
          * This event always occurs before any other during the update.
          */
-        public void begin();
+        default void begin()
+        {}
```

Comment:
super nit: Not a big deal, but I mostly see `default blah blah {}` w/ braces on the same line elsewhere in the codebase